src/main/java/http/helper/Messages.java

```diff
@@ -0,0 +1,33 @@
+package http.helper;
+
+import java.net.ServerSocket;
+
+public class Messages {
+    public static String listeningForConnection() {
+        return "Listening for connection...";
+    }
+    public static String connectionSuccessful() {
+        return "Connection Successful";
+    }
+    public static String listenForInput() {
+        return "Listening for input";
+    }
+    public static String successful(ServerSocket serverSocket) {
+        return "***Successful*** " + serverSocket;
+    }
+
+    public static String echoReceived (String inputLine) {
+        return "http.Server received : " + inputLine;
+    }
+
+    public static String bye() {
+        return "bye";
+    }
```

Comment:
This is me being a bit picky, but up until this point all the methods in this class are separated by a line of whitespace, then these last three aren't. Try to be consistent with formatting.